app/views/generic_files/_show_actions.html.erb

```diff
@@ -19,7 +19,7 @@
           <%= link_to "Feature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'feature'} %>
         <% elsif @generic_file.featured? %>
           &nbsp;|&nbsp;
-          Featured
+          <%= link_to "Unfeature", sufia.featured_work_path(@generic_file, format: :json), data: {behavior: 'unfeature-page'} %>
```

Comment:
You can create a js.erb file that will translate the i18n in the asset pipeline.  Here is an example of a js.erb file: https://github.com/projecthydra/sufia/blob/master/app/assets/javascripts/sufia/single_use_link.js.erb#L13